File: flightReservationServer/Model/src/main/java/model/Ticket.java
package model;

import java.util.Objects;

public class Ticket {

    private int id;
    private int clientId;
    private int flightId;
    private String touristFirstName;
    private String touristLastName;

    public Ticket(int id, int clientId, int flightId, String touristFirstName, String touristLastName) {
        this.id = id;
        this.clientId = clientId;
        this.flightId = flightId;
        this.touristFirstName = touristFirstName;
        this.touristLastName = touristLastName;
    }

    public Ticket(int clientId, int flightId, String touristFirstName, String touristLastName) {
        this.clientId = clientId;
        this.flightId = flightId;
        this.touristFirstName = touristFirstName;
        this.touristLastName = touristLastName;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getClientId() {
        return clientId;
    }

    public void setClientId(int clientId) {
        this.clientId = clientId;
    }

    public int getFlightId() {
        return flightId;
    }

    public void setFlightId(int flightId) {
        this.flightId = flightId;
    }

    public String getTouristFirstName() {
        return touristFirstName;
    }

    public void setTouristFirstName(String touristFirstName) {
        this.touristFirstName = touristFirstName;
    }

    public String getTouristLastName() {
        return touristLastName;
    }

    public void setTouristLastName(String touristLastName) {
        this.touristLastName = touristLastName;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Ticket ticket = (Ticket) o;
        return id == ticket.id &&
                clientId == ticket.clientId &&
                flightId == ticket.flightId &&
                Objects.equals(touristFirstName, ticket.touristFirstName) &&
                Objects.equals(touristLastName, ticket.touristLastName);
    }

    @Override
    public int hashCode() {
        return Objects.hash(id, clientId, flightId, touristFirstName, touristLastName);
    }
}

File: FlightReservationClient/Model/Ticket.cs
using System.Collections.Generic;

namespace FlightReservations.model
{
    public class Ticket
    {
        public int Id;
        public int ClientId;
        public int FlightId;
        public string TouristFirstName;
        public string TouristLastName;

        public Ticket(int id, int clientId, int flightId, string touristFirstName, string touristLastName)
        {
            Id = id;
            ClientId = clientId;
            FlightId = flightId;
            TouristFirstName = touristFirstName;
            TouristLastName = touristLastName;
        }

        public Ticket(int clientId, int flightId, string touristFirstName, string touristLastName)
        {
            ClientId = clientId;
            FlightId = flightId;
            TouristFirstName = touristFirstName;
            TouristLastName = touristLastName;
        }

        public override bool Equals(object obj)
        {
            var ticket = obj as Ticket;
            return ticket != null &&
                   Id == ticket.Id &&
                   ClientId == ticket.ClientId &&
                   FlightId == ticket.FlightId &&
                   TouristFirstName == ticket.TouristFirstName &&
                   TouristLastName == ticket.TouristLastName;
        }

        public override int GetHashCode()
        {
            var hashCode = -577858635;
            hashCode = hashCode * -1521134295 + Id.GetHashCode();
            hashCode = hashCode * -1521134295 + ClientId.GetHashCode();
            hashCode = hashCode * -1521134295 + FlightId.GetHashCode();
            hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(TouristFirstName);
            hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(TouristLastName);
            return hashCode;
        }

        public override string ToString()
        {
            return "Id:" + Id + ";ClientId:" + ClientId + ";FlightId:" + FlightId + ";TouristFirstName:" + TouristFirstName + ";TouristLastName:" + TouristLastName;
        }
    }
}
